http/play/src/main/scala/play.scala

```diff
@@ -26,26 +26,15 @@ import _root_.play.api.mvc._
 import _root_.play.api.http._
 
 object play {
-  object FreeSAction {
-    def apply[F[_]](prog: FreeS[F, Result])(
-        implicit MF: Monad[Future],
```

Comment:
Just wondering: could we loose this implicit `Monad[Future]` parameter if we `import cats.instances.future._`?